src/objective-c/tests/BUILD

```diff
@@ -188,7 +185,7 @@ grpc_objc_testing_library(
     deps = [
         "InteropTests-lib",
         "//src/objective-c:grpc_objc_client_core_cronet_testing",
-        "//test/core/end2end:end2end_tests",
```

Comment:
These 2 seems to be the only 2 references to `end2end_tests`. Does this means we're losing some test coverage not only for Cronet, but some other end to end code paths too?

Reply:
`//test/core/end2end:end2end_tests` prior to this PR was a library that exported about 100 end2end tests for gRPC.

Here they were being instantiated with a fixture supplied by CoreCronetEnd2EndTests.mm (also deleted) to test Cronet in a variety of situations against a gRPC chttp2 backend.

I think other testing of Objective-C is using other frameworks.

`cq_verifier` here is a utility class that aids in writing tests against the C-core framework.

Reply:
Ah ok! Thanks for explaining! sgtm then :)